src/pages/home/ReportScreen.js

```diff
@@ -163,6 +164,14 @@ class ReportScreen extends React.Component {
         }
 
         const reportID = getReportID(this.props.route);
+
+        const isArchivedRoom = ReportUtils.isArchivedRoom(this.props.report);
+        let archiveReason = '';
+        if (isArchivedRoom) {
+            const lastClosedActionIndex = _.findLastIndex(this.props.reportActions, action => action.type === CONST.REPORT.ACTIONS.TYPE.CLOSED);
+            archiveReason = lastClosedActionIndex >= 0 ? this.props.reportActions[lastClosedActionIndex].originalMessage.reason : CONST.REPORT.ARCHIVE_REASON.MANUALLY_ARCHIVED;
```

Comment:
I see manually_archived is kind of like the default reason right? I don't think we don't use this in practice in Auth, but maybe this makes more sense as the default action in [Auth for this function](https://github.com/Expensify/Auth/blob/8c9992b7c4fb98ac86a0a7f90d0309147fb0f59c/auth/lib/Account.h#L483)?

cc: @marcaaron

Reply:
For some context, I chose `MANUALLY_ARCHIVED` because it has the most generic UI message:

> This chat room has been archived.

Reply:
That's why it made sense to use as a fallback 🤷

Reply:
Ah, good question...

Here are the reasons I added recently...

https://github.com/Expensify/Auth/blob/1905fee2f0d21427163b335e146509c5a39f1cfa/auth/lib/Report.cpp#L125-L127

I think @sketchydroide will add some more. 

My 2 cents... if we don't find the reason then that seems like a sign that something is wrong in Auth and maybe we should log an alert or something in addition to the fallback? These safe fallbacks are nice, but sometimes hide bugs because the fallback works a little _too_ well and no one complains.

Reply:
> My 2 cents... if we don't find the reason then that seems like a sign that something is wrong in Auth and maybe we should log an alert or something in addition to the fallback?

Good suggestion!